client/templates/course/meta/meta.less

```diff
@@ -0,0 +1,5 @@
+.course-author,
+.course-added-date {
```

Comment:
Consider renaming this to align with the [createdDate](http://schema.org/dateCreated) property of the schema.org [CreativeWork](http://schema.org/CreativeWork).